### PR TITLE
fix(pie-button): DSW-000 add missing close to tag

### PR DESCRIPTION
**Pull request by xander-marjoram**
## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: packages/components/pie-button/src/index.ts [hunk 1]
```diff
@@ -223,7 +223,7 @@ export class PieButton extends LitElement implements ButtonProps {
         return html`
                     <pie-spinner
                         size="${spinnerSize}"
-                        variant="${spinnerVariant}"
+                        variant="${spinnerVariant}">
```
[the remark this answers is not in the document]
Comment (xander-marjoram) [reply packages/components/pie-button/src/index.ts:226]: Indeed! I'm testing how `snapit` behaves when you don't have one 😄

**Review by FayeCarter — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**